fsw/inc/lc_msgdefs.h

```diff
@@ -361,7 +371,7 @@
 
 #ifndef LC_OMIT_DEPRECATED
 #define LC_SET_AP_PERMOFF_CC LC_SET_AP_PERM_OFF_CC
-#define LC_ACTION_NOT_USED   LC_APSTATE_NOT_USED
```

Comment:
@jphickey note this additional change due to the rebase to incorporate your recent changes in https://github.com/nasa/LC/pull/74